src/main/java/com/checkmarx/sdk/utils/scanner/client/AstClientHelper.java

```diff
@@ -559,4 +565,82 @@ private synchronized String createProject(String projectName) {
 
         return projectId;
     }
+    
+    /**
+     * @param repoInfo may represent an actual git repo or a presigned URL of an uploaded archive.
+     * @param sourceLocation
+     */
+    protected AstScanStartHandler getScanStartHandler(RemoteRepositoryInfo repoInfo, SourceLocationType sourceLocation) {
+        log.debug("Creating the handler object.");
+
+        try {
+            HandlerRef ref = getBranchToScan(repoInfo);
+            URL effectiveUrl = repoInfo.getUrl();
+            
+            String username = StringUtils.defaultString(repoInfo.getUsername());
+            String password =  StringUtils.defaultString(repoInfo.getPassword());;
+            String credentailsType = "";
```

Comment:
1) typo
```suggestion
            String credentialsType = "";
```

2) what is the meaning of credentialsType as an empty string?

3) Do we still need to support the case when SCM credentials are embedded into repo URL?